j2v/utils/helpers.py

```diff
@@ -8,7 +8,7 @@ def get_dimension_types(dim_val):
     """
     json_type = "string"
     dim_type = "string"
-    if type(dim_val) == str and is_str_timestamp(dim_val):
+    if type(dim_val) == str and (is_str_timestamp(dim_val) or is_ISO_Time(dim_val)):
```

Comment:
At some point we might want to use [dateutil](https://dateutil.readthedocs.io/en/stable/parser.html#dateutil.parser.isoparse) which will simplify our code. However, for the time being, we can keep this dependency out of this library until we hit a limit with this approach.